AFQ/tests/test_api.py

```diff
@@ -735,10 +736,37 @@ def test_AFQ_data_waypoint():
         t1_path)
     shutil.copy(t1_path, t1_path_other)
 
+    vista_folder = op.join(
+        bids_path,
+        "derivatives/vistasoft/sub-01/ses-01/dwi")
+    freesurfer_folder = op.join(
+        bids_path,
+        "derivatives/freesurfer/sub-01/ses-01/anat")
+    lv1_fname = op.join(
+        freesurfer_folder,
+        "sub-01_ses-01_desc-LV1_anat.nii.gz")
+    if not op.exists(lv1_fname):
+        r = get(op.join(
```

Comment:
I think it would be cleaner to make a fetcher for this in the data module.

Reply:
done